Sources/JavaStdlib/JavaLangReflect/generated/Type.swift

```diff
@@ -6,4 +6,13 @@ import CSwiftJavaJNI
 public struct Type {
   @JavaMethod
   public func getTypeName() -> String
+
+  @JavaMethod
+  public func toString() -> String
```

Comment:
Need to properly fix this for all interfaces https://github.com/swiftlang/swift-java/issues/430

understanding what's going on without being able to get toString of the java types is quite a problem, i'll tackle it next. This is obviously going to be thrown away (the edit in the generated source)